lib/routers/establishment/projects.js

```diff
@@ -311,6 +315,31 @@ const refreshProject = (req, res, next) => {
     .catch(next);
 };
 
+router.get('/ras-due',
+  (req, res, next) => {
+    const { Project } = req.models;
+
+    const query = Project.query()
+      .where({ establishmentId: req.establishment.id })
+      .where('raDate', '<=', moment().add(1, 'month').toISOString())
+      .whereNull('raGrantedDate')
+
+    Promise.all([
+      query
+        .count()
+        .first()
+        .then(result => parseInt(result.count), 10),
```

Comment:
I think this is redundant now

Reply:
not sure I follow?

Reply:
Counts are automatically cast to integers since [this commit](https://github.com/UKHomeOffice/asl-schema/commit/94edd2fc2087d73e9a7fae42a2bd3dda793bada8)

Reply:
ooh nice